Workbooks/ADXCluster/AtResource/ResourceInsights.workbook

```diff
@@ -588,7 +588,7 @@
               ],
               "timeContextFromParameter": "TimeRange",
               "timeContext": {
-                "durationMs": 604800000
```

Comment:
@guregini this change existed in the fake link, did yo umake it? do we need it?

Reply:
This will be always overwritten  by the time filter parameter (604800000 ms == 7 days)